File: observer/Samuel Sales/DesafioObserver_Samuel/src/Revistas/Quatro_Rodas.java
package Revistas;

import agencia.Agencia;

/**
 *
 * @author Samuel Sales - Facul
 */
public class Quatro_Rodas implements ObserverEditoraAbril {

    @Override
    public void atualizar(Agencia entrada) {
        System.out.println(entrada.getNome() + ": Revista Quatro Rodas");
    }

}

File: observer/PadraoObserverII/PadraoObserverII/Revistas/QuatroRodas.cs
using PadraoObserverII.Interface;
using System;

namespace PadraoObserverII.Revistas
{
    class QuatroRodas : IObserverRevista
    {
        public void Atualizar(Agencia entrada)
        {
            Console.WriteLine(entrada.Nome + ": Notícias da Revista Quatro Rodas!");
        }
    }
}
